MCServer/Prefabs/Villages/AlchemistVillage.cubeset

```diff
@@ -0,0 +1,3647 @@
+
+-- AlchemistVillage.cubeset
+
+-- Defines the prefabs in the group AlchemistVillage
+
+-- NOTE: This file has been generated automatically by GalExport!
+-- Any manual changes will be overwritten by the next automatic export!
+
+
+
+
+Cubeset =
+{
+	Metadata =
+	{
+		CubesetFormatVersion = 1,
+		ExportDate = "2015-06-20 10:17:14",
+		["AllowedBiomes"] = "Desert, DesertM",
+		["IntendedUse"] = "Village",
```

Comment:
This is exported by the GalExport plugin.
The first two values are always added by the plugin, so it knows that it doesn't need to quote the values. All the remaining values (`AllowedBiomes` and `IntendedUse` here, but anything is supported) are copied directly from the DB and there's no guarantee they won't need quoting.